bundle/manifests/operator-certification-operator.clusterserviceversion.yaml

```diff
@@ -252,7 +329,10 @@ spec:
                 - --leader-elect
                 command:
                 - /manager
-                image: quay.io/opdev/operator-certification-operator:latest
+                env:
+                - name: GIT_REPO_PATH
+                  value: /git
+                image: quay.io/opdev/operator-certification-operator@sha256:27645327eff32530c2b7fcbebd9e4fa05290014a2141a2f66b7b5b1edbc7bdb4
```

Comment:
Since latest gets overwritten, this sha would not always be in quay. Seeing if we should do another release before this PR is merged.